### PR TITLE
make it possible to run the check-config script as a pre-commit hook

### PR DESCRIPTION
**Pull request by marten-seemann**
Let's do it the fancy way ;)

### CODE REVIEW

File: .githooks/pre-commit [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+tmp=$(mktemp)
+git show :config.json > $tmp
+status=0
```
Comment (marten-seemann) [inline .githooks/pre-commit:5]: I wish there was a way to just say `defer rm $tmp`.

Comment (mvdan) [reply .githooks/pre-commit:5]: you technically can via a trap, but it's not particularly pretty either: http://redsymbol.net/articles/bash-exit-traps/

**Review by mvdan — APPROVED** [review]
I wonder if this is a bit overkill, but it can't hurt :)

**Comment by marten-seemann** [comment]
> I wonder if this is a bit overkill, but it can't hurt :)

It definitely is ;)

**Pull request merged**